### PR TITLE
adding movie filter

### PR DESCRIPTION
**Pull request by 1stIndominus**
[DEMO LINK](https://1stIndominus.github.io/react_movies-list-filter/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -16,16 +25,18 @@ export const App: React.FC = () => {
 
             <div className="control">
               <input
+                value={query}
                 type="text"
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={(event) => setQuery(event.target.value)}
```
Comment (loralevitska) [inline src/App.tsx:33]: you can optimize it and move function to variable;
So 
const handleChangeQuery = (event) => setQuery(event.target.value)
onChange={handleChangeQuery}

File: src/App.tsx [hunk 2]
```diff
@@ -1,9 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = [...moviesFromServer].filter(movie => {
+    const movieTitle = movie.title.toLowerCase().includes(query.toLowerCase());
```
Comment (loralevitska) [inline src/App.tsx:9]: it's repeatable `query.toLowerCase()`. You can move it to variable and reuse

**Review by loralevitska — CHANGES_REQUESTED** [review]
almost done!

File: src/App.tsx [hunk 3]
```diff
@@ -1,9 +1,24 @@
...
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const handleChangeQuery
+    = (event: React.ChangeEvent<HTMLInputElement>) => setQuery(
+      event.target.value,
+    );
```
Comment (Anastasiia-Svintsova) [inline src/App.tsx:11]: Hard to read it
```suggestion
  const handleChangeQuery = (event: React.ChangeEvent<HTMLInputElement>) => {
    setQuery(event.target.value);
  };
```

**Review by Anastasiia-Svintsova — APPROVED** [review]

**Pull request closed without merging**